### PR TITLE
#164 #135 Button styling

### PR DESCRIPTION
**Pull request by alisonailea**
Addresses#164 AND #135
removed has-icon because we don't need it. Fixed styles for :active, :focus and created rules for Lucid icon when it is in a button.

### CODE REVIEW

File: src/components/Button/Button.less [hunk 1]
```diff
@@ -18,6 +18,14 @@
 	line-height: @Button-size-lineHeight;
 	padding: 0 @Button-size-padding;
 
+	& &-content {
+		display: flex;
+    justify-content: space-between;
```
Comment (jondlm) [inline src/components/Button/Button.less]: Indentation looks funky here

File: src/components/Button/Button.less [hunk 2]
```diff
@@ -79,6 +87,11 @@
 		line-height: @Button-size-lineHeight*1.5;
 		padding: 0 @Button-size-padding*1.5;
 		font-size: 1.2em;
+
+		.lucid-Icon {
```
Comment (jondlm) [inline src/components/Button/Button.less]: Nice, seems like a good solution!

**Comment by jondlm** [comment]
Could you push up this branch to docspot with `gulp docs-upload`? Also, be sure to run `npm test` before pushing to make sure linting/tests pass.

**Comment by jondlm** [comment]
image

**Comment by alisonailea** [comment]
manually merged with squash

**Pull request closed without merging**